src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+const ADDBOOK = 'ADDBOOK';
+const REMOVEBOOK = 'REMOVEBOOK';
```

Comment:
- [ ] Please, kindly follow the Ducks pattern for the value of each action like this:

```JAVASCRIPT
const ADD_BOOK = 'bookStore/books/ADD_BOOK';
const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
```

That way, we will meet this requirement 💪